installation.rst

```diff
@@ -165,15 +165,11 @@ The upgrade shouldn't affect the installed packages or cache information. If the
 
 Python 2 Deprecation Notice
 ---------------------------
-Before 1.6, all Conan features are fully supported in both Python2 and Python3.
-For Conan 1.6 and beyond, all features built prior to 1.6 will continue to be fully tested
-in Python 2 and Python3 moving forward, and the 'default' expectation will be to test new
-features in both Python 2 and Python 3.  However, where a new feature wishes to make use of a
-feature available in Python 3 or more easily available in Python 3, those features will be
-implemented and tested only in Python 3, and versions of Conan using Python 2 will not have
-access to that feature. Such features will be clearly documented in code and documentation.
-
-If and when Conan 2.x is released (Not expected in 2018) the level of compatibility with
-Python 2 may be reduced further.
+
+All features of Conan until version 1.6 are fully supported in both Python 2 and Python 3. However, new features in upcoming Conan releases
+that are only available in Python 3 or more easily available in Python 3 will be implemented and tested only in Python 3, and versions of
+Conan using Python 2 will not have access to that feature. This will be clearly described in code and documentation.
+
+If and when Conan 2.x is released (Not expected in 2018) the level of compatibility with Python 2 may be reduced further.
```

Comment:
Shall we consider adding something in this line: _We encourage you to upgrade to Python 3 as soon as possible. If this is impossible for you or your team, we would like to know so Conan can still be used in your organization._ (and add a mail or some way of contact).

⬆️ @memsharded, feedback, please.

Reply:
> If this is impossible for you or your team, we would like to know so Conan can still be used in your organization. 

"If this is impossible for you or your team, we would like to know it. Please give feedback at https://github.com/conan-io/conan/issues/3334,  or info@conan.io"